bindgen/codegen/helpers.rs

```diff
@@ -90,103 +88,136 @@ pub(crate) fn blob(ctx: &BindgenContext, layout: Layout) -> TokenStream {
         Some(ty) => ty,
         None => {
             warn!("Found unknown alignment on code generation!");
-            "u8"
+            syn::parse_quote! { u8 }
         }
     };
 
-    let ty_name = Ident::new(ty_name, Span::call_site());
-
     let data_len = opaque.array_size(ctx).unwrap_or(layout.size);
 
     if data_len == 1 {
-        quote! {
-            #ty_name
-        }
+        syn::parse_quote! { #ty_name }
```

Comment:
why do you need to `parse_quote!` the `ty_name` node again?

Reply:
Right, doesn't need to re-quoted now.